modin/logging/config.py

```diff
@@ -0,0 +1,100 @@
+import logging
+import datetime as dt
+import os
+import uuid
+import platform
+import psutil
+import pkg_resources
+import threading
+import time
+from modin.config import LogMode
+
+__LOGGER_CONFIGURED__: bool = False
+
+
+class MyFormatter(logging.Formatter):
+    converter = dt.datetime.fromtimestamp
+
+    def formatTime(self, record, datefmt=None):
+        ct = self.converter(record.created)
+        if datefmt:
+            s = ct.strftime(datefmt)
+        else:
+            t = ct.strftime("%Y-%m-%d %H:%M:%S")
+            s = "%s,%03d" % (t, record.msecs)
+        return s
+
+
+def get_size(bytes, suffix="B"):
+    """
+    Scale bytes to its proper format
+    e.g:
+        1253656 => '1.20MB'
+        1253656678 => '1.17GB'
+    """
+    factor = 1024
+    for unit in ["", "K", "M", "G", "T", "P"]:
```

Comment:
`get_size(1_000_000_000_000_000_000_000)` is `None` because it's past the last prefix.

Reply:
Added a case for Exabyte, but do you mean to show this example as just a case we should hardcode if above a certain value?

Reply:
Yes, I think we should hardcode beyond a certain value. I don't think we'll ever hit this bug in practice, but I prefer to fix it for tidiness.